examples/app.py

```diff
@@ -157,29 +168,41 @@
 InvenioRecordsUI(app)
 
 
-@app.cli.group()
-def fixtures():
-    """Command for working with test data."""
+app.register_blueprint(accounts_blueprint)
 
 
 def create_object(bucket, file_name, stream):
     """Object creation inside the bucket using the file and its content."""
     obj = ObjectVersion.create(bucket, file_name, stream=stream)
     rec_uuid = uuid4()
-    provider = RecordIdProvider.create(object_type='rec', object_uuid=rec_uuid)
+
+    provider = RecordIdProvider.create(
+        object_type='rec', object_uuid=rec_uuid
+    )
+
     data = {
         'pid_value': provider.pid.pid_value,
         'files': [
             {
-                'uri': '/files/{0}/{1}'.format(str(bucket.id), file_name),
+                'bucket': str(bucket.id),
                 'key': file_name,
                 'size': obj.file.size,
-                'bucket': str(bucket.id),
-                'local': True
+                'checksum': obj.file.checksum,
+                'version_id': str(obj.version_id),
+                'links':{
```

Comment:
The original code was written before invenio-records-files was done, so I think this part here can be made significantly simpler. I can help you Tuesday when I'm back in the office.